recipes/linecache2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{%set name = "linecache2" %}
+{%set version = "1.0.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "4b26ff4e7110db76eeb6f5a7b64a82623839d595c2038eeda662f2a2db78e97c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - inspect_fodder2.patch  # [not py3k]
+    - test_linecache.patch  # [not py3k]
+    # inspect_fodder2 kills the build when byte-compiling on py 2.x
+    # the patch removes inspect_fodder2 and refs to it
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr
+
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/testing-cabal/linecache2
+  license: PSF 1.0
+  summary: 'Backport of the linecache module'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
Yes, I'm fine with being added as a maintainer - or not, if anyone has some objection.